### DISCUSSION TITLE
PnP Powershell

### DISCUSSION THREAD

**Discussion by samimiashkan** [pnp/powershell#2015] [Q&A]
Hi.. I installed PNP.POWERSHELL and most of the cmdlets are working.. bit couple of cmdlets are not found.. in my case:

Copy-PnPFolder : The term 'Copy-PnPFolder' is not recognized as the name of a cmdlet, function, script file, or operable program. Check 
the spelling of the name, or if a path was included, verify that the path is correct and try again.

I can connect and use most of the cmdlets as mentioned.. I cannot figure out, how I can get the missing cmdlets.

Any idea?

Thanks..

Ashkan.

**Comment by AndersRask** [comment]
-did you install the module in PowerShell 5.1 or 7.2 and is that the same version you are testing it in? Try cmdlets in both versions if you have them installed (use $psversiontable if you are not sure what version you have)

It could also be an old SharePointPowerShellOnline module installed that interfere that only have some of the cmdlets.

 Inside PowerShell try and type:
`
gcm Get-PnPSite | fl *
`
to see what module the cmdlet belongs to. 

You can also check in the command below to see what folders the module is installed in:
`$env:PSModulePath -split ";"`

**Reply by samimiashkan** [reply]
Hi Anders.. Many thanks for response.. This helped me to find the orphan PNP install.. I removed it and reinstalled .. Now I can find all cmdlets.  Many thanks.. Much appreciated.
